frontend/src/components/events/atoms/flex/flex.css

```diff
@@ -0,0 +1,32 @@
+.flex {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  margin: 20px;
+}
+
+.flex-item {
+  display: flex;
+  flex: 0 1 27%;
+  justify-content: center;
+  align-items: center;
+  margin: 12px 8px;
```

Comment:
lets make this `20px 10px` to have be a bit more spaced out. 

then lets add `margin-top: 0px` underneath so its not double done.